packages/decentraland-ecs/src/decentraland/Events.ts

```diff
@@ -121,6 +126,10 @@ export function _initEventObservables(dcl: DecentralandInterface) {
           onPlayerExpressionObservable.notifyObservers(event.data as IEvents['playerExpression'])
           return
         }
+        case 'videoEvent': {
+          onVideoEvent.notifyObservers(event.data as IEvents['videoEvent'])
+          return
+        }
```

Comment:
I think we should look for the component by Id and reflect the changes in the component itself. We may keep the event tho.

Reply:
That way the code may become:

```ts
const video = new VideoTexture()
entity.setComponent(video)
entity.setComponent(new PlaneShape)

class MySystem {
  update(){
    if (video.position > 10) {
      // do something
    }
  }
}

```